lib/pages/chat_list/chat_list_header.dart

```diff
@@ -1,35 +1,24 @@
+import 'package:fluffychat/pages/chat_list/chat_list.dart';
 import 'package:fluffychat/pages/chat_list/chat_list_header_style.dart';
-import 'package:fluffychat/presentation/enum/chat_list/chat_list_enum.dart';
-import 'package:fluffychat/presentation/model/chat_list/chat_selection_actions.dart';
 import 'package:fluffychat/widgets/twake_components/twake_header.dart';
 import 'package:flutter/material.dart';
 
 class ChatListHeader extends StatelessWidget {
-  final ValueNotifier<SelectMode> selectModeNotifier;
-  final ValueNotifier<List<ConversationSelectionPresentation>>
-      conversationSelectionNotifier;
-  final VoidCallback openSelectMode;
   final VoidCallback? onOpenSearchPage;
-  final VoidCallback onClearSelection;
+  final ChatListController controller;
```

Comment:
Not have a better solution? Why not use directly from `MatrixState`

Reply:
Hm, remove it and create parameters